examples/advanced_tensorflow/pyproject.toml

```diff
@@ -12,6 +12,6 @@ authors = ["The Flower Authors <hello@flower.dev>"]
 
 [tool.poetry.dependencies]
 python = ">=3.7,<3.11"
-flwr = "^0.19.0"
+flwr = "^1.0.0"
 # flwr = { path = "../../", develop = true }  # Development
```

Comment:
Should we remove this line from all the examples? I'm in favor of removing

Reply:
I am fine either way. I think its useful but we could also move it into the developer docs. Wdyt?

Reply:
Cool, let's move it to the contributor docs then